docusaurus.config.js

```diff
@@ -39,18 +39,33 @@ module.exports = {
       {
         id: "optimize",
         path: "optimize",
+        includeCurrentVersion: false,
         routeBasePath: "optimize",
         beforeDefaultRemarkPlugins: [versionedLinks],
-        sidebarPath: require.resolve("./optimize_sidebars.js"),
+        sidebarPath: require.resolve(
+          "./optimize_versioned_sidebars/version-3.7.0-sidebars.json"
+        ),
         editUrl: "https://github.com/camunda/camunda-platform-docs/edit/main/",
+        lastVersion: "3.7.0",
```

Comment:
tells docusaurus that optimize v3.7.0 is the version that shouldn't include a number in the URL.